apps/skilavottord/ws/src/app/modules/accessControl/accessControl.resolver.ts

```diff
@@ -1,44 +1,47 @@
-import { Inject } from '@nestjs/common'
 import { Query, Resolver, Args, Mutation } from '@nestjs/graphql'
 
-import type { Logger } from '@island.is/logging'
-import { LOGGER_PROVIDER } from '@island.is/logging'
+import { Authorize, AuthService, AuthUser, CurrentUser } from '../auth'
 
-import { Authorize } from '../auth'
 import { AccessControlModel } from './accessControl.model'
 import { AccessControlService } from './accessControl.service'
+import {
+  UpdateAccessControlInput,
+  CreateAccessControlInput,
+} from './accessControl.input'
 
+@Authorize({ throwOnUnAuthorized: false })
```

Comment:
Apparently we can do `@Authorize({ throwOnUnAuthorized: false, role: 'developer' })`

Forgot about that one.